src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+export const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state) => {
+      state.value -= 1;
+    },
```

Comment:
- [ ] You're trying to update a value property in your `addBook` reducer, but there is no `value` property in your initial state.  you're decreasing `state.value`, which doesn't seem related to books. Kindly fix it.

- [OPTIONAL] I highly recommend you add the unique identifier `id` property to each newly added book which will help you to manipulate your books easily.

```suggestion
  addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 600),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    }
```